app/jobs/labor/rate_limit_checker_job.rb

```diff
@@ -0,0 +1,10 @@
+module Labor
+  class RateLimitCheckerJob < ApplicationJob
+    queue_as :labor_rate_limit_checker
```

Comment:
I would call the queue `rate_limit_checker` without "labor" which is more an organizational thing than anything else